OpenRA.Mods.Common/Widgets/Logic/ConnectionLogic.cs

```diff
@@ -124,7 +124,8 @@ public ConnectionFailedLogic(Widget widget, ModData modData, OrderManager orderM
 			widget.Get<LabelWidget>("CONNECTING_DESC").GetText = () => connectingDescText;
 
 			var connectionError = widget.Get<LabelWidget>("CONNECTION_ERROR");
-			var connectionErrorText = modData.Translation.GetString(orderManager.ServerError) ?? connection.ErrorMessage ?? modData.Translation.GetString(UnknownError);
+			var errorMessage = orderManager.ServerError != null ? modData.Translation.GetString(orderManager.ServerError) : connection.ErrorMessage;
```

Comment:
```suggestion
			var errorMessage = (orderManager.ServerError != null ? modData.Translation.GetString(orderManager.ServerError) : connection.ErrorMessage)
				?? modData.Translation.GetString(UnknownError);
```

?

Reply:
I wanted to make it a bit easier to parse/read, but can change it if you insist.

Reply:
I think its hard to read in both solutions

Reply:
Though I find it easier with my proposal. With the current method you are left wondering wether both of these values are used somewhere else in the constructor

Reply:
Inlined it again.